tiledb/sm/array/array_directory.h

```diff
@@ -45,11 +45,13 @@ using namespace tiledb::common;
 namespace tiledb {
 namespace sm {
 
+/** Mode for the ArrayDirectory class. */
 enum class ArrayDirectoryMode {
-  DEFAULT,
-  SCHEMA_ONLY,
-  COMMITS,
-  VACUUM_FRAGMENTS
+  DEFAULT,          // Default mode.
```

Comment:
Maybe call this `READ`? Does default mean load all URIs? (can be addressed in a future PR)